config/locales/views/campagnes.yml

```
@@ -49,6 +49,8 @@ fr:
           zero: Aucune évaluation
           one: Consulter 1 évaluation
           other: Consulter %{count} évaluations
+        nouvelle_campagne_disabled: Nouvelle campagne
+        nouvelle_campagne_disabled_explication: La création de nouvelles campagnes a été désactivée par le compte “admin” de votre structure. Vous pouvez utiliser une des campagnes existantes.
```

Avec les guillemets français. Attention de vérifier que github a bien aussi ajouté les espaces insécables.
```suggestion
        nouvelle_campagne_disabled_explication: La création de nouvelles campagnes a été désactivée par le compte « admin » de votre structure. Vous pouvez utiliser une des campagnes existantes.
```